DESCRIPTION

```diff
@@ -27,9 +27,12 @@ Depends:
     ospsuite (>= 11.0)
 Imports:
     FME,
-    rClr,
+    ggplot2,
+    ggtext,
```

Comment:
@Felixmil @svavil  Any idea why I had to explicitly import `ggtext` here? Without doing so, tests failed with 

```
== Failed tests ================================================================
-- Error ('test-parameter-identification.R:101:3'): Plotting works before running the parameter estimation task --
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'ggtext'
Backtrace:
     x
  1. +-vdiffr::expect_doppelganger("before_estimation", task$plotResults()[[1]]) at test-parameter-identification.R:101:2
  2. | \-vdiffr (local) writer(fig, testcase, title)
  3. |   \-vdiffr:::print_plot(plot, title)
  4. +-task$plotResults()
  5. | \-base::lapply(...)
  6. |   \-ospsuite.parameteridentification (local) FUN(X[[i]], ...)
  7. |     \-ospsuite::plotIndividualTimeProfile(dataCombined[[idx]], plotConfiguration)
  8. |       \-ospsuite:::.plotTimeProfile(dataCombined, defaultPlotConfiguration)
  9. |         \-tlf::plotTimeProfile(...)
 10. |           \-tlf:::.setPlotObject(plotObject, plotConfiguration)
 11. |             +-plotObject %||% initializePlot(plotConfiguration)
 12. |             \-tlf::initializePlot(plotConfiguration)
 13. |               \-tlf::setPlotLabels(plotObject)
 14. |                 \-labels$updatePlot(newPlotObject)
 15. |                   +-ggplot2::theme(...)
 16. |                   | \-ggplot2:::find_args(..., complete = NULL, validate = NULL)
 17. |                   |   \-base::mget(args, envir = env)
 18. |                   \-private$.xlabel$createPlotTextBoxFont()
 19. |                     \-self$font$createPlotTextBoxFont(...)
 20. \-base::loadNamespace(x)
 21.   \-base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 22.     \-base (local) withOneRestart(expr, restarts[[1L]])
 23.       \-base (local) doWithOneRestart(return(expr), restart)

```

Reply:
I would also expect that it wouldn't require an explicit import. Might there be a requirement in `{tlf}` that wasn't explicitly listed as import?